src/peeker/config.toml

```diff
@@ -10,7 +10,7 @@
 [[sources]]
 kafka_broker = "kafka:9092"
 schema_registry = "http://schema-registry:8081"
-topic_namespace = "mysql.tpcch."
+topic_namespace = "debezium.tpcch."
```

Comment:
For my information: it's worth calling out that we'll need to specify the peeker version to run if we want to run load tests across this commit boundary. 

I don't think that there are any changes you can make to make this better.